operatorconfig/moduleconfig/observability/v1.10.0/karavi-metrics-powerflex.yaml

```diff
@@ -105,7 +105,7 @@ spec:
       serviceAccount: karavi-metrics-powerflex-controller
       containers:
         - name: karavi-metrics-powerflex
-          image: <POWERFLEX_OBS_IMAGE>
+          image: dellemc/csm-metrics-powerflex:v1.7.0
```

Comment:
What is the reason for changing the image parameter placeholder to the image name??

Reply:
It is handled the same way for all other components.